test/test_translator.py

```diff
@@ -155,6 +218,23 @@ def test_get_road_direction():
 
     assert test_direction==valid_direction
 
+def test_get_road_direction_invalid_direction():
+    test_coordinates = ''
+    test_direction=icone_translator.get_road_direction(test_coordinates)
+    valid_direction= []
+    assert test_direction==valid_direction
+
+    test_coordinates = []
+    test_direction=icone_translator.get_road_direction(test_coordinates)
+    valid_direction= []
+    assert test_direction==valid_direction
+
+    test_coordinates = None
+    test_direction=icone_translator.get_road_direction(test_coordinates)
+    valid_direction= []
+    assert test_direction==valid_direction
+
+def test_get_road_direction_valid_direction():
```

Comment:
Can we split these unit tests out to test a single unit of logic? IE we should have a test for northbound, a separate test for southbound, eastbound, and westbound.

Reply:
I did break down those unit test and renamed the function name with specific direction name